### DISCUSSION TITLE
v1.8.1

### DISCUSSION THREAD

**Discussion by aluminum-ice** [aluminum-ice/pwnagotchi#80] [Announcements]
## What's Changed
* Merge pull request #1 from aluminum-ice/master by @fernandofatech in https://github.com/aluminum-ice/pwnagotchi/pull/65
* Config plugins defaults by @retiolus in https://github.com/aluminum-ice/pwnagotchi/pull/66
* Update ntfy.py to support custom notifications title, message and tags by @retiolus in https://github.com/aluminum-ice/pwnagotchi/pull/68
* CI: Fixed and Improved by @shashankx86 in https://github.com/aluminum-ice/pwnagotchi/pull/64
* add Waveshare V4 driver and integration by @marvellousmarvolo in https://github.com/aluminum-ice/pwnagotchi/pull/75
* add delay between bettercap and pwnagotchi starts by @noncenz in https://github.com/aluminum-ice/pwnagotchi/pull/76

## New Contributors
* @fernandofatech made their first contribution in https://github.com/aluminum-ice/pwnagotchi/pull/65
* @shashankx86 made their first contribution in https://github.com/aluminum-ice/pwnagotchi/pull/64
* @marvellousmarvolo made their first contribution in https://github.com/aluminum-ice/pwnagotchi/pull/75
* @noncenz made their first contribution in https://github.com/aluminum-ice/pwnagotchi/pull/76

**Full Changelog**: https://github.com/aluminum-ice/pwnagotchi/compare/1.8.0...v1.8.1

<hr /><em>This discussion was created from the release <a href='https://github.com/aluminum-ice/pwnagotchi/releases/tag/v1.8.1'>v1.8.1</a>.</em>

**Comment by Uhrien** [comment]
Can we please have a step by step guide on how to install/configure it on a Raspberry Pi Zero 2W with a Waveshare v4 screen?
I'm flashing the firmware via Raspberry Pi Imager, waiting for it to flash, then creating the config.toml file inside the microsd and pasting this:

main.name = "pwnagotchi"
main.lang = "en"
main.whitelist = [
  "MY_HOMENETWORK"
]

main.plugins.grid.enabled = true
main.plugins.grid.report = true
main.plugins.grid.exclude = [
  "YourHomeNetworkHere"
]

ui.display.enabled = true
ui.display.type = "waveshare_3"
ui.display.color = "black"

ui.web.username = "admin"
ui.web.password = "admin"
ui.web.enabled = true
ui.web.address = "0.0.0.0"
ui.web.origin = ""
ui.web.port = 8080
ui.web.on_frame = ""

ui.fps = 1

ai.enabled = true



 But when I stick my microsd back in my RPI all I get is some "flashes" on my screen and then it becomes half grey, half black and get not output.
 
I've also installed the RNDIS driver on W10 (and the device is read as a USB Ethernet/RNDIS Gadget) and I've configured its network settings like so:
![image](https://github.com/aluminum-ice/pwnagotchi/assets/39272420/a51acbc8-3521-4026-b07c-2260b6cd236f)


I can SSH into it, but it won't let me access through its webgui at 10.0.0.2:8080.
And as I've already mentioned, the screen is just dead :D

What am I doing wrong? What's the best way I can troubleshoot this? ^_^

Also, this is a picture of what the screen looks like:

![image](https://github.com/aluminum-ice/pwnagotchi/assets/39272420/43fee83f-fe2f-4f98-bf6c-1057c82c78c5)

**Reply by Uhrien** [reply]
Little update on this topic:

My screen was apparently faulted. I ordered another one (that's also V4) and it works now.

**Reply by MagicalMutation** [reply]
had the same issue...  tried everything and so many different images.

bought another screen and worked straight away..  feel like a right idot..

one thing i noted was the faulty one was slightly differnet in that the little square in the bottom of the screen (by the pull tab for the screen protector)  was faded compared to the working one.  

just got to figgure out now how to get the keys out of the capured data.
